README.md

```diff
@@ -110,6 +110,54 @@ curl localhost:8080
 # Output: Hello World from a PHP HTTP function!
 ```
 
+## Run your function on Google Cloud Functions
+
+Follow the steps below to deploy to Google Cloud Functions. More information
+on function deployment is available in the
+[GCF documentation](https://cloud.google.com/functions/docs/deploying).
+
+To run your function on Cloud Functions, first you must have the [gcloud SDK][gcloud] installed and [authenticated][gcloud-auth].
+
+Additionally, you need to have a Google Cloud project ID for the
+[Google Cloud Project][gcp-project] you want to use.
+
+> **Note:** PHP support on Cloud Functions is currently in limited alpha.
+> It is not yet suitable for production workloads, and support is best-effort
+> only. Access is currently limited to selected early-access users.
+
+Make sure your source file (which defines your function) is called
+`index.php`. The Functions Framework lets you choose a function source file,
+but Cloud Functions currently uses the default of `index.php`.
+
+Decide _which_ function in the source file to invoke, that is, the name that you
+used when writing the function. This is called the **target**.
+
+Choose a Cloud Functions **name** for your function. The **name** identifies
+this function deployment (e.g. in the cloud console) and is also part of the
+function's default URL. (Note: the **name** and the **target** do not have to
+be the same value.)
+
+Then, from the directory containing your function source, issue the gcloud command to deploy:
+
+```sh
+gcloud functions deploy $YOUR_FUNCTION_NAME \
+    --project=$YOUR_PROJECT_ID \
+    --runtime=php74 \
+    --trigger-http \
+    --entry-point=$YOUR_FUNCTION_TARGET
```

Comment:
- `--project=` is not necessary as we'll use the current gcloud project. This flag is used if you want to deploy a function in a different project, which is rare.
- nit: `--trigger-http` is usually the last flag for a deploy command. Usually it's deploy, runtime, entry-point, trigger.

Reply:
`--project`: The instructions do note that it can be omitted if deploying to the current default project. Or would you prefer I remove it?
`--trigger-http`: Done

Reply:
Please remove `--project` to match our Cloud Function docs. (`cs/en/functions/docs/ "--project"`)

We do not include optional [global flag](https://cloud.google.com/sdk/gcloud/reference) overrrides in the docs.

Reply:
Sounds good. Removed.